examples/scenarioSepMomentumManagement.py

```diff
@@ -97,7 +97,6 @@
 
 import os
```

Comment:
Was this file really meant to be changed as part of this update and are these changes necessary?  On the surface of things, they look unrelated and unnecessary.

Reply:
This file was changed to remove some code that is now depreciated.  That is why it is part of this update.